release-notes/swan-lake-beta1-release-note.md

```diff
@@ -41,10 +41,33 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 ### Language Updates
 
-#### New Features
-
 #### Improvements
 
+- Worker deadlock detection has been improved to include the `wait` action.
+- `self` of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is not a subtype of `readonly` or `isolated object`.
+- `xml:strip()` does not mutate the XML value, thus using `xml:strip()` on immutable XML values is allowed.
+
+#### Breaking Changes
+
+- An included record parameter of a function can only be specified after any required and/or defaultable parameters of the function.
+- Additive expressions and multiplicative expressions are no longer supported with numeric values when the static types of the operands belong to different numeric basic types.
+- Configurable variables are implicitly `final` now. Moreover, the type of such a variable is now effectively the intersection of the specified type and `readonly`. Therefore configurable variables no longer support the `final` and `isolated` qualifiers.
+- Type narrowing will no longer take place for captured variables of an anonymous function, since the narrowed type cannot be guaranteed during the execution of the function.
```

Comment:
```suggestion
- Type narrowing will no longer take place for captured variables of an anonymous function since the narrowed type cannot be guaranteed during the execution of the function.
```